test/client_test.cc

```diff
@@ -30,6 +30,12 @@ TEST_F(ClientTest, NormalRun) {
 }
 
 TEST_F(ClientTest, AutoConcurrencyRun) {
+  // This works around an error thrown by TCLAP about multiple unlabeled optional args not being
```

Comment:
Maybe just add a pointer to the other comment and move this higher up in the file, e.g.  
```c++
// See the comment in TestUtility::createOptionsImpl()
````

Reply:
Commented both occurrences here with the pointer. What did you mean we should move?

Reply:
Sorry that remark about moving didn't make sense, thanks for updating this, marking as resolved.